### PR TITLE
Add build locally script

### PR DESCRIPTION
**Pull request by techcobweb**
Signed-off-by: Mike Cobbett <77053+techcobweb@users.noreply.github.com>

### CODE REVIEW

File: build-locally.sh [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+    public-private gpg pair. eg: For development you could use your signing github
+    passphrase.
+
+EOF
+}
+
+#-----------------------------------------------------------------------------------------                   
+# Process parameters
+#-----------------------------------------------------------------------------------------                   
+exportbuild_type=""
```
Comment (eamansour) [inline build-locally.sh:89]: Needs a space between "export" and "build_type", though this variable isn't being used.

**Review by eamansour — APPROVED** [review]

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

**Pull request merged**